### PR TITLE
Create GET /admins endpoint

### LINKED ISSUE #419: Create new GET /admins endpoint accessible to admins only
## Description

As a admin of the app,
I need to have a List Admins endpoint,
so that I can get the list of fellow admins.

Admins should have quick access to other admins. E.g.: in case they wish to remove an admin.

Create a new endpoint `GET /admins`, only accessible to Admin users, that lists all users who are admin (excluding the user accessing the API). This should appear under the Admins namespace on Swagger UI.

Please, commit your change using our [Commit message Style Guide](https://github.com/systers/mentorship-backend/wiki/Commit-Message-Style-Guide).

### Update [Required]
- [ ] Create a new endpoint GET /admins
- [ ] Write at least one test showing your changes are working.

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
3 hours

### PR DESCRIPTION
**Pull request by naveennvrgup**
### Description
Added admins/ API and tests for the same.

Fixes #419

### Type of Change:

- Code
- Documentation

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Added unit tests.

image

### Checklist:

- [X] My PR follows the style guidelines of this project
- [X] I have performed a self-review of my own code or materials
- [X] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [X] My changes generate no new warnings 
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/api/resources/admin.py [hunk 1]
```diff
@@ -76,3 +77,37 @@ def post(cls):
...
+
+
+@admin_ns.route('admins')
+class ListAdmins(Resource):
+
+    @classmethod
+    @jwt_required
+    @admin_ns.doc('get_list_of_admins')
+    @admin_ns.doc(responses={
+        401: f"{messages.TOKEN_HAS_EXPIRED['message']}<br>"
```
Comment (isabelcosta) [inline app/api/resources/admin.py:89]: can you also document error 403 you are returning in the end of this method?
here's an example: https://github.com/anitab-org/mentorship-backend/pull/459/files#diff-40b144b4232d9ee0bc35b84eb4226c7cR52

File: app/api/dao/admin.py [hunk 2]
```diff
@@ -87,3 +87,19 @@ def revoke_admin_user(user_id, data):
...
+            user_id: The ID of the user querying the fellow admins.
+        
+        Returns:
+            A list of admin users matching conditions and the HTTP response code.
+        """
+
+        users_list = UserModel.query.filter(UserModel.id != user_id).all()
+        list_of_users = [user.json() for user in users_list if user.is_admin]
+
+        return list_of_users, 200
```
Comment (isabelcosta) [inline app/api/dao/admin.py]: can you add a blank line in the end?

File: app/api/models/admin.py [hunk 3]
```diff
@@ -11,3 +11,43 @@ def add_models_to_namespace(api_namespace):
         description='The unique identifier of a user'
     )
 })
+
+
+public_admin_user_api_model = Model('User list model', {
```
Comment (isabelcosta) [inline app/api/models/admin.py:17]: shouldn't you do the same for this model, as in line 5 of this file?

File: app/api/dao/admin.py (hunk 2 above)
Comment (isabelcosta) [inline app/api/dao/admin.py]: if you are returning on admin.py resource the 200, why do you need to send it here?
You don't do anything with this result there https://github.com/anitab-org/mentorship-backend/pull/459/files#diff-40b144b4232d9ee0bc35b84eb4226c7cR107

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@naveennvrgup please address my change requests, and let me know if you have an opinion on them :)
Thank you for your contribution!

File: app/api/resources/admin.py (hunk 1 above)
Comment (naveennvrgup) [reply app/api/resources/admin.py]: I have now added docs for 200 and 403

File: app/api/dao/admin.py (hunk 2 above)
Comment (naveennvrgup) [reply app/api/dao/admin.py]: Sure

Comment (naveennvrgup) [reply app/api/dao/admin.py]: oops! i have fixed the code now

File: app/api/models/admin.py (hunk 3 above)
Comment (naveennvrgup) [reply app/api/models/admin.py]: Sure. fixed it

**Review by naveennvrgup — COMMENTED** [review]
Done the required fixes. Please review again

File: app/api/models/admin.py (hunk 3 above)
Comment (naveennvrgup) [reply app/api/models/admin.py:17]: Sure. Fixed it

File: app/api/resources/admin.py (hunk 1 above)
Comment (naveennvrgup) [reply app/api/resources/admin.py:89]: I have now added docs for 200 and 403

File: app/api/dao/admin.py (hunk 2 above)
Comment (naveennvrgup) [reply app/api/dao/admin.py]: sure

Comment (naveennvrgup) [reply app/api/dao/admin.py]: Sorry! I have fixed it now

**Review by isabelcosta — APPROVED** [review]
I just tested and it all seems fine! Thank you for your contribution @naveennvrgup :)

**Pull request merged**